tests/testthat/test-class-Megastudy.R

```diff
@@ -368,7 +423,7 @@ test_that("imputeZeroes method is sane", {
   expect_equal(nrow(imputedDT[imputedDT$sample.specimen_count == 0]), 0)
 
   # multiple special vocabs in same plot, w one shared weighting var
```

Comment:
Isn't this the same as the test starting at line 155?
Both have species, count, sex and attractant.
Ah, not quite. In the earlier one, sex was "not plotted".